apps/scan/frontend/src/screens/diagnostics_screen.tsx

```diff
@@ -137,6 +141,7 @@ export function DiagnosticsScreen({
         mostRecentAudioDiagnostic={
           mostRecentAudioDiagnosticQuery.data ?? undefined
         }
+        markThresholds={systemSettings.markThresholds}
```

Comment:
Do we have a process for updating the `readiness-report` [mocks](https://github.com/votingworks/vxsuite/blob/main/apps/scan/backend/src/__image_snapshots__/readiness-report-1.png) that we use for testing?

Reply:
First confirm that the diff output is what you expected. The path is in the test output: `apps/scan/backend/src/__image_snapshots__/__diff_output__/readiness-report-1-diff.png`

Then to update the snapshot run `npm run test:ci -- -u`, or run the test and press `u` in the watcher tool.

Reply:
Awesome, thanks for the clear instructions. It looks as expected, so I've pushed the updated snapshot

Reply:
No problem. Some of this information is in the test output, though that output is admittedly noisy.